doc/manual/manual.tex

```diff
@@ -320,11 +320,12 @@ \subsection{Referencing \aspect{}}
 }
 \end{lstlisting}
 
+and cIte the  most recent release of  the software package. See \url{https://aspect.dealii.org/publications}.
```

Comment:
An empty line in latex indicates a paragraph break. So if you want the sentence to start before the lstlisting, you need to remove the empty line.